_posts/2019-08-28-foreman-birthday-party-at-netways.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: post
+title: 'Foreman's 10th Birthday Party Celebration @NETWAYS'
+date: 2019-08-28 18:19:42
+author:
+ name: Rahul Bajaj
+ uri: rabajaj
+tags:
+- foreman
+- event
+- meetup
+- birthday
+- community
+---
+
+A quick recap of the Forman birthday event which includes new community member’s introductions, productive conversation, interesting statistics, lots of swag and cake!
+
+<!-- more -->
+
+## Prelude
+
+The Foreman birthday event was a community driven, a one-day celebration. It was held at the NETWAYS office in Nuremberg on 25th July 2019. The event majorly consisted of 2 hours of hacking and demo followed by multiple talks. Let me take you through them in more detail in this blog.
+The birthday event aimed to present a perfect opportunity of networking with the community and explore the modern trends and solutions in the Foreman project. Connecting the community, discussing & debating on latest technologies were one of the main focus areas of the event.
+
+# The Event
+
+The party started at 12:30 pm with a hacking session. A few demo machines were prepared by Dirk for users to explore Foreman while our community folks from ATIX had a booth where they demonstrated the working of orcharhino.
+
+During the hacking session, there were many productive discussions that took place on topics such as the Foreman documentation, string translations in Foreman core and plugins, Foreman salt plugin and many more. There were a few issues that were discovered and filed to redmine on similar topics.
+
+## Talks and presentations:
+
+1) A community talk:
+
+Ohad Levy, the founder of the Foreman project opened the day sharing some interesting stories as well as pictures of the Foreman project’s journey since the initial days.
+
+<p float="left" align='center'>
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image1.JPG" width="200" />
+</p>
+
+After Ohad, Greg Sutcliffe demonstrated the community growth in the last few years using his new exceptional statistical skills, presenting data in forms of animated charts and graphs. Ori Rabin followed Greg and spoke about the current community structure and she also spoke about the results extracted from the community survey. Lastly, I spoke about how we would grow the community in the future.
+
+<p float="left" align='center'>
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image2.JPG" width="200" />
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image3.JPG" width="200" />
+  <img src="/static/images/blog_images/2019-birthday-celebration-images/image4.JPG" width="200" />
+</p>
+
+2) The State of Debian Support
+
+Quirin Pamp spoke about the state of Debian support in the Foreman community.
+
```

Comment:
I wanted to add few more lines here.

Reply:
@quba42 feel free to add few lines here :)

Reply:
I may get around to it tomorrow.

Reply:
I could imagine something like the following:
```
Quirin Pamp spoke about "The State of Debian Support in Foreman/Katello".
The talk focused on Debian/Ubuntu package actions in Katello (now merged), as well as Debian/Ubuntu errata support (undergoing active review here: https://github.com/Katello/katello/pull/7961).
The latter included some discussion of the ATIX errata service to obtain the needed Debian/Ubuntu errata (https://github.com/ATIX-AG/errata_parser, https://github.com/ATIX-AG/errata_server).
```
Feel free to make adjustments as you see fit.
(Or would you rather I create a pull request against your issue branch? :stuck_out_tongue_winking_eye: )

Reply:
Thanks for your input here :) I have added the lines mentioned by you :)